src/config/defaults.js

```diff
@@ -1,4 +1,5 @@
 export const defaults = {
   locale: process.env.DEFAULT_LOCALE || 'en',
-  etvasGraphQLUrl: process.env.ETVAS_GRAPHQL_URL
+  etvasGraphQLUrl: process.env.ETVAS_GRAPHQL_URL,
+  connect: false
```

Comment:
Let's do not confuse the consumer about the `connect` function with the `connect` config directive:

```suggestion
  oidc: false // if true, please refer to connect functionality
```